modules/recording/recording/dashboard.tsx

```diff
@@ -94,24 +128,28 @@ const Dashboard = () => {
       messageHandler.handleError(t("recording.select-file-upload"));
     }
   };
+
   return (
     <div className="p-5">
       <h2 className={dash_styles.h2}>{t("recording.record-title")}</h2>
-      <p className={`${dash_styles.p} ${dash_styles.top_p}`}>
-        {t("recording.activate-audio")}
-      </p>
+      <p className={dash_styles.p}>{t("recording.activate-audio")}</p>
 
       <div className={dash_styles.contentColumns}>
         <div className={dash_styles.recordSection}>
           <MicrophoneSelect value={microphone} onChange={setMicrophone} />
           <button
-            className={`${dash_styles.recordButton} ${recordingState === "recording"
-              ? dash_styles.recording
-              : recordingState === "paused"
-                ? dash_styles.paused
-                : ""
-              }`}
+            className={`${dash_styles.recordButton} ${
+              recordingState === "recording"
+                ? dash_styles.recording
+                : recordingState === "paused"
+                  ? dash_styles.paused
+                  : ""
+            }`}
             onClick={toggleRecording}
+            style={{
+              transform: `scale(${1 + audioLevel * 0.4})`,
+              transition: "transform 0s ease-in-out",
+            }}
```

Comment:
You addeda. alot of logic for this audio button

He can be promoted to be his own component, keeping dashboard clean and readible.